src/UITests/LoadEventFile.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.IO;
+using System.Reflection;
+
+namespace UITests
+{
+    /// <summary>
+    /// Start the app, open an A11yEvent file, spot check loaded data, exit app
+    /// </summary>
+    [TestClass]
```

Comment:
@jalkire , Should these tests use the "Integration" category you introduced?

Reply:
Yes, except that the Integration category is part of axe-windows. I suggested adding TestCategories in the same style to ai-win in a previous PR, but we haven't gotten around to it yet